From 795637c218ca474affb35c75b8300b64b6e5943d Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 11 May 2022 14:10:37 +0200
Subject: [PATCH 197/954] feat(stm32mp1-fdts): change name of encrypted key OTP

Alignment with OP-TEE device tree for the name NVMEM cell in BSEC node
of the encrypted key OTPs.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 155775cb9f8ae90b361976b4a0f171809b46fa00 feat(stm32mp1-fdts): change name of encrypted key OTP]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ic7bb552b8ddda1baa64160f5a05d8f413ea45de4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/251860
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp131.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp131.dtsi b/fdts/stm32mp131.dtsi
index 8bcf363b6f..2ed27374d9 100644
--- a/fdts/stm32mp131.dtsi
+++ b/fdts/stm32mp131.dtsi
@@ -454,7 +454,7 @@
 				reg = <0xe4 0xc>;
 				st,non-secure-otp;
 			};
-			enckey_otp: enckey_otp@170 {
+			oem_enc_key: oem_enc_key@170 {
 				reg = <0x170 0x10>;
 			};
 		};
-- 
2.53.0

